Dshop/apps/products_catalogue/filters.py

```diff
@@ -0,0 +1,10 @@
+from django_filters import rest_framework as filters
+from .models import Product
+
+
+class ProductFilter(filters.FilterSet):
+    name = filters.CharFilter(field_name='name', lookup_expr='icontains')
```

Comment:
Please double check if it will filter out not active products.

Reply:
@jacoor in api_views.py I have in queyset added filter: `queryset = Product.objects.filter(is_active=True, parent_product=None)` this doesn't take care of it?

Reply:
Should. However, adding is_active here would improve readability.

Reply:
done